src/hott/08-families-of-maps.rzk.md

```diff
@@ -16,109 +16,98 @@ maps.
   ( A : U)
   ( B C : A → U)
   ( f : (a : A) → (B a) → (C a))
-  : ( Σ (x : A) , B x) → (Σ (x : A) , C x)
-  := \ z → (first z , f (first z) (second z))
+  : ( total-type A B) → (total-type A C)
+  := \ (a , b) → (a , f a b)
 
-#def total-map-to-fiber
+#def fib-total-map-fib-fiberwise
   ( A : U)
   ( B C : A → U)
   ( f : (a : A) → (B a) → (C a))
-  ( w : (Σ (x : A) , C x))
-  : fib (B (first w)) (C (first w)) (f (first w)) (second w) →
-    ( fib (Σ (x : A) , B x) (Σ (x : A) , C x) (total-map A B C f) w)
-  :=
-    \ (b , p) →
-      ( (first w , b) ,
-        eq-eq-fiber-Σ A C (first w) (f (first w) b) (second w) p)
+  ( (a , c) : total-type A C)
+  : fib (B a) (C a) (f a) (c)
+  → fib (total-type A B) (total-type A C) (total-map A B C f) (a , c)
+  := \ (b , p) → ((a , b) , eq-eq-fiber-Σ A C a (f a b) c p)
 
-#def total-map-from-fiber
+#def fib-fiberwise-fib-total-map
   ( A : U)
   ( B C : A → U)
   ( f : (a : A) → (B a) → (C a))
-  ( w : (Σ (x : A) , C x))
-  : fib (Σ (x : A) , B x) (Σ (x : A) , C x) (total-map A B C f) w →
-    fib (B (first w)) (C (first w)) (f (first w)) (second w)
+  : ( (a , c) : total-type A C)
+  → fib (total-type A B) (total-type A C) (total-map A B C f) (a , c)
```

Comment:
Same as above, `map-total` for the functorial action on maps of sigmas, or `map-Σ`. However, there's also the binary action to consider. In `agda-unimath` we call `map-total` just `tot`, but I think `map-total` is better. The binary action is called `map-Σ`

Reply:
That's reasonable, I just figured this was an appropriate occasion to bring it up.